Packs/Base/ReleaseNotes/1_33_9.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### CommonServerPython
+
+Fixed an issue where **TableToMarkdown** Failed to parse Non JSON serializable objects.
```

Comment:
If TableToMarkdown is a command, then it should be with 3 asterisks (***TableToMarkdown***)